index.html

```diff
@@ -0,0 +1,456 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars hambuger" id="hamburger"></i>
+      </button>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <aside class="menu-bar">
+      <ul>
+        <li class="mobile-lis"><a href="#about-me">About</a></li>
+        <li class="mobile-list"><a href="#projects">Portfolio</a></li>
+        <li class="mobile-list"><a href="#skillset">Skills</a></li>
+        <li class="mobile-list"><a href="#contact">Contact</a></li>
+      </ul>
```

Comment:
- [ ] The previous reviewer forgot to mention that you must follow the given [Formspree service](https://formspree.io/html). :1st_place_medal: 
As you can see on this image, your options position is a little bit changed. Please fix this issue.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/69011963/142658812-1a2298fd-01e0-4f62-b5e8-d96a3521242a.png) | ![scrnli_10_8_2021_6-04-05 PM](https://user-images.githubusercontent.com/69011963/142686851-d46d2f1b-6336-4cbe-bb75-88f0e7426bdf.png) |